README.md

```diff
@@ -1,2 +1,66 @@
 # members_only
-A microverse project to work with aunthentcation in rails
+A microverse project to work with authentcation in rails, creating posts and restricting non_users from access to view posts.
+
+
+
+## Built With
+
+- Ruby on Rails
+- VSCode
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+1. Ruby v.2.7.0
+2. Rails 6.0.3.
+
+### Setup
+
+1. Clone this repository by using the Terminal to navigate to the desired location on your computer and then enter the following line of code into the Terminal:
+```
+git clone https://github.com/mimipeshy/re-former.git
+```
+2. Run `cd members_only` to move inside the downloaded repository.
+3. Run `bundle install` to download the project's dependencies.
+4. Run `rails db:migrate` to set up your local database.
+5. Run `rails server` and navigate to http://localhost:3000/users/new in a web browser to view the user creation form.
+6. To register as a user navigate to: http://localhost:3000/users/sign_up.
+7. After registering you can create posts.
+
+
```

Comment:
##  Improvement  on the commit messages
- [ ] In your next iteration, I would urge to commit as often as posbblie to make it easy to follow the development journey let's aim to make our commits more descriptive ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))